css/selectmenu.css

```diff
@@ -146,6 +146,18 @@
 	box-shadow: 0 1px 2px rgba(0, 0, 0, 0.07) inset;
 	color: #32373c;
 }
+/* Override to have same styles as WP form styles since WordPress 5.4 */
```

Comment:
That's the jQuery UI styles that you are overriding, right?

Reply:
Specifically Polylang selectmenu styles overwrite form's WordPress styles because jQuery UI selectmenu widget generate a `span` HTML tag which isn't target by WordPress form's styles which target only HTML standard form tags (input... , select, textarea ... )
So we need to target specifically jQuery UI selectmenu widget in the Polylang settings languages  page.

Reply:
As long as WordPress doesn't use a select library. For the reference: https://core.trac.wordpress.org/ticket/31696 was planned for 4.1, then 4.4 then the project focused on other priorities ;-)